File: bronze/baekjoon_1145.java
import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    int [] input = new int[5];
	    for(int i = 0; i < 5; i++){
	        input[i] = sc.nextInt();
	    }
	    
	    int temp = 1;
	    while(true){
	        int cnt = 0;
	        for(int i = 0; i < 5; i++){
	            if(temp % input[i] == 0){
	                cnt++;
	            }
	        }
	        
	        if(cnt >= 3){
	            break;
	        }
	        temp++;
	    }
	    
	    System.out.print(temp);
	    
	}
}

File: bronze/baekjoon_1145.kt
fun main() {
   val nums = readLine()!!.split(" ").map{it.toInt()}
  
  var temp = 1
   while(true) {
		var cnt = 0
		
		for(i in 0..4){
			if(temp % nums[i] == 0) 
				cnt++
		}
		if(cnt > 2 )
			break
		temp++
   }
   
   print(temp)
   
}